tests/unit/test_solvers/test_idaklu_jax.py

```diff
@@ -100,6 +102,10 @@ def test_instantiate_fails(self):
     not pybamm.have_idaklu() or not pybamm.have_jax(),
     "IDAKLU Solver and/or JAX are not available",
 )
+@pytest.mark.skipif(
+    sys.platform.startswith("win"),
+    reason="Problem with tests crashing workers on Windows",
+)
 class TestIDAKLUJax(unittest.TestCase):
```

Comment:
Why wasn't this needed before?

Reply:
It was, but we never added it. 
>Have a look this [job](https://github.com/cringeyburger/PyBaMM/actions/runs/10536847176/job/29197492807#step:9:3760). I turned on IDAKLU build for Windows. There was no problem with building it, but the tests failed because of the workers crashing. I tried the same on my Windows laptop, and it's even worse.
Even the current development branch doesn't test for IDAKLU on Windows, so I think we might have to spend a bit more time, and guessing by the amount of work needed, it's better tackled on another fresh PR.

Also
>Not sure if you know this, but if you click on the codacy message, it opens a window that gives a detailed explanation of what it means and how you can fix it.

I know about this and checked it too, but it was kinda vague 😅.

Reply:
If the skip test was needed before, then why weren't the failing tests showing up in CI before we used sbc? That is what I am trying to understand. What changed